### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by fewensa**
Related: 
- #998 
- #1004 

Preview:
https://github.com/fewensa/darwinia/releases/tag/v10.10.12
https://github.com/fewensa/darwinia/releases/tag/pango-v10.10.12

test new docker image

```
docker run --rm -it --name cab ghcr.io/darwinia/darwinia:sha-7bd4944 --chain=crab
```

### CODE REVIEW

File: .github/release-template-dev.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+#### Pull with the Git Commit SHA
+```docker
+docker pull ghcr.io/darwinia-network/darwinia:sha-{{ sha }}
+```
+
+## Proposal Hashes
+
+|        -        |                Pangolin                |                Pangoro                |
+|:---------------:|:--------------------------------------:|:-------------------------------------:|
```
Comment (AurevoirXavier) [inline .github/release-template-dev.md:32]: suggest:
```
          hash1 hash2 hash3
pangolin
pangoro
```

File: .maintain/docker/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+ARG USERNAME=darwinia
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+COPY dist/ /home/$USERNAME/darwinia-nodes/
+
+RUN groupadd --gid $USER_GID $USERNAME \
+    && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
+    && apt-get update && apt-get install -y --no-install-recommends sudo ca-certificates \
+    && echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME \
```
Comment (AurevoirXavier) [inline .maintain/docker/Dockerfile:12]: Why we must use sudo here?

**Comment by AurevoirXavier** [comment]
Let's re-format this.

**Comment by fewensa** [comment]
All done. format like https://github.com/fewensa/darwinia/releases/tag/v10.10.12

**Review by AurevoirXavier — CHANGES_REQUESTED** [review]
We should build testnets and mainnets separately.

We shouldn't use `require: [build-package]`.

**Pull request merged**